### PR TITLE
Document searchDynamicUrls

### PR DESCRIPTION
**Pull request by ekoleda-codaio**
[Preview](https://coda-packs-sdk--2572.com.readthedocs.build/en/2572/guides/blocks/sync-tables/dynamic/#search)

### CODE REVIEW

File: docs/images/dynamic_sync_table_search.png
Comment (jonathan-codaio) [inline docs/images/dynamic_sync_table_search.png:1]: Alicia had me fix the alignment of the search box so it lines up with the text above it (e.g. "FormResponses") to show the information hierarchy better, it should be on head already if you want to update the screenshots. Sorry!

**Review by jonathan-codaio — APPROVED** [review]
Awesome!

While you're here, mind adding an entry in the CHANGELOG for this under 1.3.3? I had intentionally omitted it because I didn't know when we were going to get the greenlight to release it.

File: docs/images/dynamic_sync_table_search.png
Comment (ekoleda-codaio) [reply docs/images/dynamic_sync_table_search.png:1]: Done.

**Comment by ekoleda-codaio** [comment]
> While you're here, mind adding an entry in the CHANGELOG for this under 1.3.3?

Done.

**Pull request merged**